Xamarin.Forms.Core/Flags.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Linq;
+using static System.String;
+
+namespace Xamarin.Forms
+{
+	public static class CoreFlags
+	{
+		internal const string CollectionViewExperimental = "CollectionView_Experimental";
```

Comment:
so, the magic string is internal, but the helper is public ?

I'd drop this class, and move those into the CollectionView class. Keep the string internal, or we won't be able to drop it in the future, and turn the helper internal or at least not EditorBrowseable

Reply:
Done.